src/pages/about.jsx

```diff
@@ -1,7 +1,22 @@
-import React from "react";
+import React from 'react';
+import aboutPageContent from '../assets/about-content.json';
+import Content from '../components/Content';
 
 const About = () => {
-	return <div></div>;
+    const content = aboutPageContent.map((aboutContent) => {
+        return (
+            <div>
+                <Content
+                    headline={aboutContent.headline}
+                    content={aboutContent.content}
+                    iframeTitle={aboutContent.tedxIframeTitle}
+                    iframeLink={aboutContent.tedxIframeLink}
+                />
+            </div>
+        );
+    });
+
+    return <div>{content}</div>;
```

Comment:
I see a lot of 'content content' which is not making the code readable to an external reader kindly do something like naming the map function const's name to `renderContent`  (referring to lines `6`& `19`)